pkgs/applications/editors/neovim/default.nix

```diff
@@ -153,4 +153,12 @@ in
       maintainers = with maintainers; [ manveru rvolosatovs ];
       platforms   = platforms.unix;
     };
-  }
+  };
+in
+  nvim.overrideAttrs(oa: {
+
+    passthru = {
+      # returns a wrapped neovim with everything needed
+      withConfig = callPackage ./wrapper.nix { } nvim;
```

Comment:
isn't this `wrapNeovimUnstable` instead of `wrapNeovim`?

Reply:
yes I would like to make wrapNeovimUnstable the default (the difference is you can pass plugins along with their config).